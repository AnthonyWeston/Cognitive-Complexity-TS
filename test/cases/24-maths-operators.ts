function plus() {
    1 + 1 + 1
}

function minus() {
    1 - 1 - 1
}

function size() {
    1 < 1
}

function equals() {
    1 === 1
}
