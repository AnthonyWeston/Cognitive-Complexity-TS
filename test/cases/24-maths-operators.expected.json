{
    "24-maths-operators.ts": {
        "score": 0,
        "inner": [
            {
                "name": "plus",
                "score": 0,
                "line": 1,
                "column": 1
            },
            {
                "name": "minus",
                "score": 0,
                "line": 5,
                "column": 1
            },
            {
                "name": "size",
                "score": 0,
                "line": 9,
                "column": 1
            },
            {
                "name": "equals",
                "score": 0,
                "line": 13,
                "column": 1
            }
        ]
    }
}
